engine/src/main/resources/org/camunda/bpm/engine/impl/mapping/entity/Authorization.xml

```diff
@@ -60,7 +60,8 @@
       RESOURCE_TYPE_ = #{resourceType, jdbcType=INTEGER},
       RESOURCE_ID_ = #{resourceId, jdbcType=VARCHAR},
       PERMS_ = #{permissions, jdbcType=INTEGER},
-      REMOVAL_TIME_ = #{removalTime, jdbcType=TIMESTAMP}
+      REMOVAL_TIME_ = #{removalTime, jdbcType=TIMESTAMP},
+      ROOT_PROC_INST_ID_ = #{rootProcessInstanceId, jdbcType=TIMESTAMP}
```

Comment:
Maybe here as well it makes sense to add the root instance id to the persistent state then.

Reply:
👍